### PR TITLE
Details popup window

### PR DESCRIPTION
**Pull request by chiztechnology**
Hello ! I just set up an interesting feature on my portfolio about preview of projects. 😄 
Please have a look on it and tell me what i can do to improve my coding skills. 🎊 
Let's notice that I still get some troubles when Showing the modal (rendering the model multiple times).
I count on your help to move forward and complete this project. 🤝 
Faithfully yours 🙇‍♂️

### CODE REVIEW

File: js/fetch_data.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  span.onclick = () => {
+    modal.style.display = 'none';
+  };
+
+  window.onclick = (event) => {
+    if (event.target === document.getElementById('modal')) {
+      modal.style.display = 'none';
+    }
+  };
+}
```
Comment (Whoistolu) [inline js/fetch_data.js]: - [ ] [REQUIRED] Kindly ensure that the desktop modal displays the rest of the modal contents as provided in the figma design. Right now I can't see the contents, I can only see the modal image. I would suggest you place the content and the image in a single div.
- [ ] [REQUIRED] Please make sure that the modal for the mobile fits properly at the center. You currently have more margin at the top of the mobile modal.
- [ ] [REQUIRED] Also, please check the modal of the the mobile version and give it two buttons (See-live and See-source) their appropriate margins as it is in the figma.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
Screenshot 2022-11-17 at 20 29 22 | Screenshot 2022-11-17 at 20 30 13

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
Screenshot 2022-11-17 at 20 28 32 | Screenshot 2022-11-17 at 20 27 46

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @chiztechnology 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/fetch_data.js (hunk 1 above)
Comment (Whoistolu) [reply js/fetch_data.js]: Hello @chiztechnology , I recommended the above changes for you while you pushed a new commit. Your linters passed initially but just as I recommended changes, you pushed new commits at the same time, hence, your stylelint is now failing. Kindly fix this and all the recommended changes before requesting another review.
### Happy coding 🍾

Comment (danmainah) [reply js/fetch_data.js]: - [ ] @team great work implementing the changes requested however, I noticed the website broke after the change was implemented, I can only access one popup only once, it requires me to reload the page to view the next popup, kindly fix it.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @chiztechnology , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chiztechnology** [comment]
I need some help to fix it. I'm still struggling on fixing that issue.
My modal works only once then require me to refresh the page to open another one. !

dear @danmainah if you've an idea on how to solve this ; it will be helpful for me

**Comment by danmainah** [comment]
> I need some help to fix it. I'm still struggling on fixing that issue. My modal works only once then require me to refresh the page to open another one. !
> 
> dear @danmainah if you've an idea on how to solve this ; it will be helpful for me

@chiztechnology, check your looping method, how it does it, then how popup window, use console.log to check the data being outputted by each action

**Comment by chiztechnology** [comment]
hey! it's working
I removed some pieces of code and wrote them in another way and finally, they were working! I feel happy
Thanks a lot 🎊 😃

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @chiztechnology  

Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kamba56 — COMMENTED** [review]
Hi @chiztechnology 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a pull request that is not for this project. Kindly submit the right pull request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**